drivers/gpu/drm/tests/drm_format_helper_test.c

```diff
@@ -530,7 +532,10 @@ static void drm_test_fb_xrgb8888_to_gray8(struct kunit *test)
 	KUNIT_ASSERT_NOT_ERR_OR_NULL(test, xrgb8888);
 	iosys_map_set_vaddr(&src, xrgb8888);
 
-	drm_fb_xrgb8888_to_gray8(&dst, &result->dst_pitch, &src, &fb, &params->clip);
+	if (result->dst_pitch == TEST_USE_DEFAULT_PITCH)
+		drm_fb_xrgb8888_to_gray8(&dst, NULL, &src, &fb, &params->clip);
+	else
+		drm_fb_xrgb8888_to_gray8(&dst, &result->dst_pitch, &src, &fb, &params->clip);
```

Comment:
Sorry for the silly question, but `drm_fb_xrgb8888_to_gray8(&dst, NULL, &src, &fb, &params->clip);` isn't the same as `drm_fb_xrgb8888_to_gray8(&dst, 0, &src, &fb, &params->clip);`?

Reply:
it is the same, but why it would be better to replace it with 0?

Reply:
I feel that it makes the code looks simpler just using the line `drm_fb_xrgb8888_to_gray8(&dst, &result->dst_pitch, &src, &fb, &params->clip);`

Reply:
Oh, I got confused with that too, but in this way, you're not passing NULL as an argument when `dst_pitch` equals 0. Because you're passing the address of `result->dst_pitch`.